app/src/substrate_types.c

```diff
@@ -112,6 +128,20 @@ parser_error_t _readBalanceOf(parser_context_t* c, pd_BalanceOf_t* v)
     return _readBalance(c, &v->value);
 }
 
+parser_error_t _readBytes(parser_context_t* c, pd_Bytes_t* v)
+{
+
```

Comment:
```suggestion
```

Reply:
odd spaces here

Reply:
yeah.. need to tune up the linter a little bit more